src/BQ24195.cpp

```diff
@@ -1195,6 +1202,68 @@ bool PMICClass::isBatteryInOverVoltage() {
     return 0;
 }
 
+
+/*******************************************************************************
+ * Function Name  : setFastChargeTimerSetting
+ * Description    : Sets the charging safety timer
+ * Input          : safety timer duration in hours (rounds up)
+ * Return         : 0 on Error, 1 on Success
+ *******************************************************************************/
+bool PMICClass::setFastChargeTimerSetting(float hours) {
+
+    int DATA = readRegister(CHARGE_TIMER_CONTROL_REGISTER);
+
+    if (DATA == -1) {
+        return 0;
+    }
+
+    byte mask = DATA & 0b11111001;
```

Comment:
Here now I'd love to have a more expressive bitmask. You seem to be masing bytes 7, 1 and 2. Which meaning does those bytes have? I suggest using names for those bits and then rewrite using bitmasks, i.e.
```C++
enum CHARGE_TIMER_CONTROL_REGISTER
{
  BIT_NAME_7 = 7,
  BIT_NAME_2 = 2,
  BIT_NAME_1 = 1,
};

DATA &= ~((1<<BIT_NAME_7) | (1<<BIT_NAME_2) | (1<<BIT_NAME_1));
```